test/tests/deepkitType.test.ts

```diff
@@ -0,0 +1,95 @@
+import { classToPlain, t, plainToClass } from '@deepkit/type';
+import { getModelForClass, prop } from '../../src/typegoose';
+
+enum Group {
+  confidential = 'confidential',
+  public = 'public',
+}
+
+class Account {
+  @t.mongoId.group(Group.public)
+  public _id: string;
```

Comment:
are you sure its `string` here? without `@prop` the actual type would be a `mongoose.Types.ObjectId`

Reply:
Yes, the @deepkit/type `mongoId` decorator seems to make it work. 

Scott

> also changed commiter info if the first commit (to be the same as the later ones)
> 
> PS: also thanks to the rebase re-commiting the commits they are now signed with my key (you are the author, i am the commiter - [see this stackoverflow question](https://stackoverflow.com/questions/18750808/difference-between-author-and-committer-in-git))

I'm still a GIt noob, so I hope your comment wasn't sarcasm and just an honest thank you. Hehehe.... I really am not sure. 

Scott

Reply:
> I'm still a GIt noob, so I hope your comment wasn't sarcasm and just an honest thank you. Hehehe.... I really am not sure.

it was not really a thank you or sarcasm, i just noted that i changed the commiter information on the original first commit and what the rebase has done to the commits (with a stackoverflow question that explains it really good)